### DISCUSSION TITLE
Not being able to open MRs with renovate

### DISCUSSION THREAD

**Discussion by rabajaj0509** [renovatebot/renovate#25918] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

"renovateVersion": "37.60.5"

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### What is your question?

In my repository, there were initially 10 MRs open due to Renovate. However, I specifically wanted to focus on security-related MRs. As GitLab does not offer the option of `vulnerabilityAlerts,` I opted for `osvVulnerabilityAlerts` instead. Consequently, I modified my renovate.json file accordingly:
```
{
    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
    "enabledManagers": ["gomod"],
    "rebaseWhen": "behind-base-branch",
    "lockFileMaintenance": {
      "enabled": true,
      "automerge": true
    },
    "packageRules": [
      {
        "enabled": false,
        "matchPackagePatterns": ["*"],
        "separateMajorMinor": false,
        "automerge": false
      }
    ],
    "osvVulnerabilityAlerts": true
  }
```

Upon updating my renovate.json, I proceeded to delete the previously created 10 branches and also revised the titles and descriptions of the closed MRs, assuming that Renovate would not recognize these changes and would generate new MRs accordingly. However, I now encounter an issue: despite my intention to test Renovate's capability to generate only OSV-related MRs, no MRs are being created. Is there an issue with my renovate.json or did I do something wrong by deleting the branches? Any help would be highly appreciated, thanks!

### Logs (if relevant)

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

**Comment by rarkins** [comment]
Your config disables every package

**Reply by rabajaj0509** [reply]
Is there a way I can get only the security-related ones? I have been following the same configurations as mentioned in this thread -- https://github.com/renovatebot/renovate/discussions/15490

**Reply by rarkins** [reply]
I'm not sure if that's possible via config today
